demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -16,52 +17,55 @@ class ReleaseNotesValidator(BaseValidator):
     """Release notes validator is designed to ensure the existence and correctness of the release notes in content repo.
 
     Attributes:
-        file_path (str): the path to the file we are examining at the moment.
+        release_notes_file_path (str): the path to the file we are examining at the moment.
         release_notes_path (str): the path to the changelog file of the examined file.
         latest_release_notes (str): the text of the UNRELEASED section in the changelog file.
         master_diff (str): the changes in the changelog file compared to origin/master.
     """
 
-    def __init__(self, file_path, modified_files=None, pack_name=None, added_files=None, ignored_errors=None,
+    def __init__(self, release_notes_file_path, modified_files=None, pack_name=None, added_files=None, ignored_errors=None,
                  print_as_warnings=False, suppress_print=False):
         super().__init__(ignored_errors=ignored_errors, print_as_warnings=print_as_warnings,
                          suppress_print=suppress_print)
-        self.file_path = file_path
+        self.release_notes_file_path = release_notes_file_path
         self.modified_files = modified_files
         self.added_files = added_files
         self.pack_name = pack_name
-        self.release_notes_path = get_release_notes_file_path(self.file_path)
+        self.release_notes_path = get_release_notes_file_path(self.release_notes_file_path)
         self.latest_release_notes = get_latest_release_notes_text(self.release_notes_path)
+        self.file_types_that_should_not_appear_in_rn = {FileType.TEST_SCRIPT, FileType.TEST_PLAYBOOK, FileType.README,
+                                                        FileType.RELEASE_NOTES, None}
 
     def are_release_notes_complete(self):
         is_valid = True
         modified_added_files = itertools.chain.from_iterable((self.added_files or [], self.modified_files or []))
         if modified_added_files:
             for file in modified_added_files:
-                if not any(permitted_type in file for permitted_type in VALIDATED_PACK_ITEM_TYPES):
+                if find_type(file) in self.file_types_that_should_not_appear_in_rn:
                     continue
                 elif self.pack_name + '/' in file:
-                    update_rn_util = UpdateRN(pack_path=self.file_path, pack_files=set(), update_type=None,
-                                              added_files=set(), pack=self.pack_name)
+                    file = UpdateRN.check_for_release_notes_valid_file_path(file)
```

Comment:
let's add a comment here that we are changing desc.md and _image.png to the `.yml` file as we want it to be mentioned in the rn.

Reply:
done